CHANGES.rst

```diff
@@ -30,6 +30,9 @@ astropy-helpers Changelog
 
 - Switch to using mathjax instead of imgmath for local builds. [#342]
 
+- Fix incorrect behavior when using ``exclude`` parameter of various functions
+  in setup_helpers. [#331]
```

Comment:
The code change suggest exclude is actually deprecated rather than fixed?

Reply:
You're right, I've clarified it in an updated entry.